staging/src/k8s.io/kubectl/pkg/cmd/create/create_deployment.go

```diff
@@ -191,6 +197,7 @@ func (o *CreateDeploymentOptions) Run() error {
 		if o.FieldManager != "" {
 			createOptions.FieldManager = o.FieldManager
 		}
+		createOptions.FieldValidation = string(o.ValidationDirective)
```

Comment:
was this file just done as an example? there look to be ~17 other create_*.go kubectl commands that follow the same pattern

Reply:
yikes, good catch. No, definitely meant to be done for all of them, I think I just missed everything because I didn't understand what heuristic I could use to ensure that I had accounted for all of them. I think I have that now, which is that this is needed for every command that uses `cmd.AddValidateFlags`

I've gone through and added it now for all of them. Ideally, I imagine we would want some kind of test that ensured we added it to all of them (instead of relying on me to say "I've done it for every command that matches this heuristic", but what I can think of (adding a new cmd-test for every command) doesn't seem all that practical.

Let me know what you think.